demo/portal/src/app/services/userService.js

```diff
@@ -41,6 +52,12 @@ UserService.prototype.logout = function logout() {
 }
 
 
-angular.module('wfm.common.apiservices').service("userService", ['authService', function(authService) {
-  return new UserService(authService);
+angular.module('wfm.common.apiservices').service("userService", ['authService', '$http', function(authService, $http) {
+  var urlPromise = new Promise(function(resolve, reject) {
```

Comment:
I don't think there's a way to set a base url for the module globally, we'd still have to concatenate strings around. i.e. `$http.defaults.baseUrl + '...'`, but I was using a promise because I couldn't think of a better way to ensure the fh sdk init had been run.

You can still reuse that code from here and not wrap fh.init again by requiring the `baseUrl` value from the `wfm.http` module: https://github.com/feedhenry-raincatcher/raincatcher-angularjs/blob/master/packages/http/baseUrl.js#L19